### ISSUE TITLE
Segmentation fault on GenerateTranslationProfile

### ISSUE THREAD

**Issue by josuebarrera** [Alistair-Turcan/iRibo#4]
Dear @Alistair-Turcan and @aaronwacholder,

Thank you for creating iRibo! I'd like to use your software to predict actively translated ORFs in a genome, but I stumbled upon a segmentation fault error while running `GenerateTranslationProfile`. Here is the error:

`/var/spool/gridengine/execd/node609/job_scripts/6399019: line 21: 20554 Segmentation fault      iRibo --RunMode=GenerateTranslationProfile --Genome=genome.fasta --Riboseq=riboseq.sorted.bam --CandidateORFs=candidate_orfs --Threads=20 --Max_Length=40`

The genome assembly is around 200 Mbp in length, while the BAM file is only 3.7 Gb in size. I ran the job using 200 and 400 Gb of RAM, but the run crashes almost instantly so I don't think memory is the issue.  Any help would be deeply appreciated.

Best regards,
Josué.

**Comment by Alistair-Turcan** [comment]
Set the Max_Length to be 39, thats the length iRibo can handle up until (and most riboseq is shorter than that).

**Comment by josuebarrera** [comment]
Dear @Alistair-Turcan,

Thank you very much for the recommendation. The segmentation fault was caused by the `Max_Length` parameter, so I kept the default value of 35. But now it gives me another obscure error:

```
terminate called after throwing an instance of 'std::out_of_range'
  what():  basic_string::substr: __pos (which is 18446744073709551613) > this->size() (which is 1)
Aborted
```

I ran the code as:

`iRibo --RunMode=GenerateTranslationProfile --Genome=genome.fasta --Riboseq=riboseq.sorted.bam --CandidateORFs=candidate_orfs`

Maybe I'm doing something wrong?

Best regards,
Josué.
